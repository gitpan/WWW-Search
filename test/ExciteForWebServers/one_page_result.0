<html><head><title>Search Results</title></head>
<body BGCOLOR="#FFFFFF" TEXT="#000000" LINK="#FF0000" ALINK="#0000FF"><h1><a href="http://www.excite.com/"><img src="/Excite/pictures/AT-search_banner.gif" BORDER=0></a></h1>
<FORM ACTION="AT-Full_Site_Searchsearch.cgi" METHOD=POST>
<INPUT TYPE=image BORDER=0 NAME="groupby" SRC="/Excite/pictures/AT-grouped_confidence.gif">
<INPUT TYPE=hidden NAME="docs" VALUE="10199|1|0 10517|1|1 10435|1|2 10139|1|3 7688|1|4 7180|1|5 749|1|6 23157|1|7 11941|1|8 11016|1|9 6376|1|10 2577|1|11 13885|1|12 867|1|13 5609|1|14 866|1|15 17240|1|16 847|1|17 13408|2|18 4755|2|19 18775|2|20 4756|2|21 2455|2|22 4754|2|23 18774|2|24 17239|2|25 4841|2|26 2569|2|27 2094|2|28 12825|2|29 13299|2|30 7694|2|31 22767|2|32 22873|2|33 22868|2|34 22774|2|35 22694|2|36 22702|2|37 16303|2|38 22213|2|39 4759|2|40 4757|2|41 4758|2|42 4764|2|43 870|2|44 6482|2|45 10134|2|46 3996|2|47 19961|2|48 18491|2|49 16361|2|50 16348|2|51 164|2|52 19960|2|53 765|2|54 62|2|55 21744|2|56 18814|2|57 13985|2|58 18968|2|59 13345|2|60 18489|2|61 304|2|62 306|2|63 305|2|64 764|2|65 2256|2|66 2263|2|67 18846|2|68 18847|2|69 18850|2|70 18849|2|71 18848|2|72 18785|2|73 8440|2|74 18212|2|75 2332|2|76 795|2|77 75|2|78 107|2|79 14667|2|80 18198|2|81 14669|2|82 18960|3|83 19003|3|84 4840|3|85 18488|3|86 18885|3|87 309|3|88 510|3|89 12897|3|90 21743|3|91 19943|3|92 7561|3|93 8367|3|94 4276|3|95 468|3|96 5605|3|97 7946|3|98 9442|3|99">
<INPUT TYPE=hidden NAME="smode" VALUE="confidence">
<INPUT TYPE=hidden NAME="mode" VALUE="concept">
<INPUT TYPE=hidden NAME="searchpage" VALUE="">
<INPUT TYPE=hidden NAME="search" VALUE="burundi">

</FORM>

<br>excite for web servers found documents about: <b>burundi</b>
<p>
<FONT SIZE=2>
<img src="/Excite/pictures/AT-red_x.gif" alt="[higher confidence]">
- higher confidence,
<img src="/Excite/pictures/AT-black_x.gif" alt="[lower confidence]">
- lower confidence; click icons to find similar documents
</FONT>

<p><A HREF="AT-Full_Site_Searchsearch.cgi?doc=d10199&"><img border=0 src="/Excite/pictures/AT-red_x.gif"></A> 62% <A architext=result HREF="/html/edi/sourcebook/sbxb13.htm"> <b>Burundi Social Action Project</b></A> <ul> <i>Summary</i>: The World Bank Participation Sourcebook.  Sourcebook Index.  Go to: Sourcebook Home Page | Sourcebook Index Return to EDI Home Page </ul>
<A HREF="AT-Full_Site_Searchsearch.cgi?doc=d10517&"><img border=0 src="/Excite/pictures/AT-red_x.gif"></A> 62% <A architext=result HREF="/html/edi/sourcebook/sbxa14.htm"> <b>Africa: Burundi</b></A> <ul> <i>Summary</i>: The World Bank Participation Sourcebook.  Sourcebook Index.  Go to: Sourcebook Home Page | Sourcebook Index Return to EDI Home Page </ul>
<A HREF="AT-Full_Site_Searchsearch.cgi?doc=d10435&"><img border=0 src="/Excite/pictures/AT-red_x.gif"></A> 62% <A architext=result HREF="/html/edi/sourcebook/sbxs11.htm"> <b>Social Action Project, Burundi</b></A> <ul> <i>Summary</i>: The World Bank Participation Sourcebook.  Sourcebook Index.  Go to: Sourcebook Home Page | Sourcebook Index Return to EDI Home Page </ul>
<A HREF="AT-Full_Site_Searchsearch.cgi?doc=d10139&"><img border=0 src="/Excite/pictures/AT-red_x.gif"></A> 62% <A architext=result HREF="/html/edi/sourcebook/sbxa0104.htm"> <b>Africa: Burundi</b></A> <ul> <i>Summary</i>: The World Bank Participation Sourcebook.  Sourcebook Index.  Go to: Sourcebook Home Page | Sourcebook Index Return to EDI Home Page </ul>
<A HREF="AT-Full_Site_Searchsearch.cgi?doc=d7688&"><img border=0 src="/Excite/pictures/AT-red_x.gif"></A> 58% <A architext=result HREF="/html/prmge/tools/afrndx.htm"> <b>AfricaIndex.htm</b></A> <ul> <i>Summary</i>:  Web site design assistance and html production provided by the Value-Added Vendor's Network, Washington, DC -- 202-965-4363. </ul>
<A HREF="AT-Full_Site_Searchsearch.cgi?doc=d7180&"><img border=0 src="/Excite/pictures/AT-red_x.gif"></A> 58% <A architext=result HREF="/html/prmge/afrndx.htm"> <b>Africa Region: Country Gender Profiles</b></A> <ul> <i>Summary</i>:  Web site design assistance and html production provided by the Value-Added Vendor's Network, Washington, DC -- 202-965-4363. </ul>
<A HREF="AT-Full_Site_Searchsearch.cgi?doc=d749&"><img border=0 src="/Excite/pictures/AT-red_x.gif"></A> 58% <A architext=result HREF="/aftdr/gender/lstfrm.htm"> <b>type_Document_Title_here</b></A> <ul> <i>Summary</i>: To find information on Household Surveys - First Choose a Country Angola Benin Burundi Burkina Faso Central African Republic C&ocirc;te d'Ivoire Ghana Guinea Niger Nigeria Sierra Leone South Africa Tanzania Uganda Zambia </ul>
<A HREF="AT-Full_Site_Searchsearch.cgi?doc=d23157&"><img border=0 src="/Excite/pictures/AT-red_x.gif"></A> 58% <A architext=result HREF="/sp/sfunds/present/html/tsld0087.htm"> <b>Experience (cont.)</b></A> <ul> <i>Summary</i>: SOCIAL PROTECTION Experience (cont.). In Europe and Central Asia, funds exist in Albania and Armenia and are under development in Romania, Moldova and Georgia. In post conflict situations they have been used in El Salvador, Nicaragua, Ethiopia and Eritrea and potentially they will be used in Angola, Rwanda and Burundi. Previous slide Next slide Back to first slide View graphic version Feedback:... </ul>
<A HREF="AT-Full_Site_Searchsearch.cgi?doc=d11941&"><img border=0 src="/Excite/pictures/AT-red_x.gif"></A> 56% <A architext=result HREF="/html/edi/iemt-pre/lopez.htm"> <b>File Name: partbio4</b></A> <ul> <i>Summary</i>:   Lopez Martinez, Antonio Antonio Lopez Martinez is Director of the Euro-Arab Management School (EAMS) in Granada, Spain and a member of the Spanish Diplomatic Service having the category of Plenipotential Minister.D. in International Law from the University of Murcia, Spain and had studied International Law in Paris. He was Ambassador for Spain in Zaire, Ruanda, Burundi, and the Congo. Martinez... </ul>
<A HREF="AT-Full_Site_Searchsearch.cgi?doc=d11016&"><img border=0 src="/Excite/pictures/AT-red_x.gif"></A> 56% <A architext=result HREF="/html/edi/iemt/lopez.htm"> <b>File Name: partbio4</b></A> <ul> <i>Summary</i>:   Lopez Martinez, Antonio Antonio Lopez Martinez is Director of the Euro-Arab Management School (EAMS) in Granada, Spain and a member of the Spanish Diplomatic Service having the category of Plenipotential Minister.D. in International Law from the University of Murcia, Spain and had studied International Law in Paris. He was Ambassador for Spain in Zaire, Ruanda, Burundi, and the Congo. Martinez... </ul>
<A HREF="AT-Full_Site_Searchsearch.cgi?doc=d6376&"><img border=0 src="/Excite/pictures/AT-red_x.gif"></A> 56% <A architext=result HREF="/html/fpd/technet/mdf/edi-trng/lopez.htm"> <b>File Name: partbio4</b></A> <ul> <i>Summary</i>:   Lopez Martinez, Antonio Antonio Lopez Martinez is Director of the Euro-Arab Management School (EAMS) in Granada, Spain and a member of the Spanish Diplomatic Service having the category of Plenipotential Minister.D. in International Law from the University of Murcia, Spain and had studied International Law in Paris. He was Ambassador for Spain in Zaire, Ruanda, Burundi, and the Congo. Martinez... </ul>
<A HREF="AT-Full_Site_Searchsearch.cgi?doc=d2577&"><img border=0 src="/Excite/pictures/AT-red_x.gif"></A> 52% <A architext=result HREF="/html/extdr/eligible.htm"> <b>CURRENT IDA-ELIGIBLE BORROWERS</b></A> <ul> <i>Summary</i>:  Vincent Congo, Democratic Republic of Vanuatu.  Ethiopia South Asia Middle East and North Africa Eritrea Afghanistan Egypt Gambia Bangladesh Yemen, Republic of Ghana Bhutan.  Malawi Europe and Central Asia.  Sao Tome and Principe Kyrgyz Republic. </ul>
<A HREF="AT-Full_Site_Searchsearch.cgi?doc=d13885&"><img border=0 src="/Excite/pictures/AT-red_x.gif"></A> 50% <A architext=result HREF="/html/extpb/abshtml/13807.htm"> <b>Structural Aspects of Manufacturing in Sub-Saharan Africa:  Findings from a Seven Country Enterprise.  World Bank Discussion Paper No. 346</b></A> <ul> <i>Summary</i>: World Bank Publication Abstract.  Publications Home Page Full-Text Search Advanced Search How to Order Title: Structural Aspects of Manufacturing in Sub-Saharan Africa: Findings from a Seven Country Enterprise. World Bank Discussion Paper No. 13807 / $7.95 / Price code 007 Assesses the findings from firm-level surveys conducted in seven countries—Burundi, Cameroon, Ghana, Kenya, Tanzania, Zambia,... </ul>
<A HREF="AT-Full_Site_Searchsearch.cgi?doc=d867&"><img border=0 src="/Excite/pictures/AT-red_x.gif"></A> 50% <A architext=result HREF="/aftdr/afcbf/staff.htm"> <b>ACBF Staff</b></A> <ul> <i>Summary</i>: ACBF Staff &nbsp; All staff can be contacted on (263-4) 738520, 790398, 790399, or 700208 &nbsp; Abel L. Thoahlane, Executive Secretary (Lesotho).org Apollinaire Ndorukwigira, Principal Program Officer (Burundi).Org Genevisi O. Ogiogio, Principal Program Officer (Nigeria).org Edwin N. Forlemu, Senior Legal and Program Officer (Cameroon).Org Jacques G. Katuala, Senior Program Officer (Democratic... </ul>
<A HREF="AT-Full_Site_Searchsearch.cgi?doc=d5609&"><img border=0 src="/Excite/pictures/AT-red_x.gif"></A> 50% <A architext=result HREF="/html/extdr/medguide/madavo.htm"> <b>The World Bank Group Media Guide</b></A> <ul> <i>Summary</i>: The World Bank Group Media Guide Callisto Madavo.  Vice President, Africa Region Expertise: Africa, development economics, urban development Mr. Mr. Media Guide | World Bank Group Home.  Doing Business with the Bank | About the World Bank Group. </ul>
<A HREF="AT-Full_Site_Searchsearch.cgi?doc=d866&"><img border=0 src="/Excite/pictures/AT-red_x.gif"></A> 50% <A architext=result HREF="/aftdr/afcbf/proj.htm"> <b>Project Summaries</b></A> <ul> <i>Summary</i>:  ACBF Portfolio </ul>
<A HREF="AT-Full_Site_Searchsearch.cgi?doc=d17240&"><img border=0 src="/Excite/pictures/AT-red_x.gif"></A> 48% <A architext=result HREF="/html/opr/ois/casp.html"> <b>CAS Database Query</b></A> <ul> <i>Summary</i>:   Annex A2.  Annex A2 - IFC and MIGA Program.  Annex A3.  Annex A7.  Annex A7 - IFC. </ul>
<A HREF="AT-Full_Site_Searchsearch.cgi?doc=d847&"><img border=0 src="/Excite/pictures/AT-red_x.gif"></A> 48% <A architext=result HREF="/aftdr/particip/teams.htm"> <b>teams.htm</b></A> <ul> <i>Summary</i>: Paula Donnelly-Roark Central African RepublicEllen CohenRudi Chizungu Chad.. GambiaRogerio Pinto.. KenyaIan KnappKhaled Sherif LesothoRoss Paul. ZambiaRoss Paul. ZimbabweRogerio Pinto. </ul>
<A HREF="AT-Full_Site_Searchsearch.cgi?doc=d13408&"><img border=0 src="/Excite/pictures/AT-black_x.gif"></A> 48% <A architext=result HREF="/html/extpb/abshtml/12994.htm"> <b>Policies Affecting Fertility and Contraceptive Use: An Assessment of Twelve Sub-Saharan Countries.  World Bank Discussion Paper No. 259.</b></A> <ul> <i>Summary</i>: World Bank Publication Abstract. World Bank Discussion Paper No. 12994 / $7.95 / Price code 007 Reviews family planning activities and ranks countries according to the strength of their policies regarding girls' schooling, child health, and women's legal status. However, levels of contraceptive use are still low, and the family size desired by couples remains high. The author reviews family... </ul>
<A HREF="AT-Full_Site_Searchsearch.cgi?doc=d4755&"><img border=0 src="/Excite/pictures/AT-black_x.gif"></A> 48% <A architext=result HREF="/html/extdr/offrep/afr/afrid97t.htm"> <b>AFRIDA</b></A> <ul> <i>Summary</i>: Africa Region: IDA Subscriptions of Voting Power, and Subscriptions and Contributions 1997.  of total Subscriptions and. Source: 1997 World Bank Annual Report.  Africa | World Bank Group Home.  Doing Business with the Bank |About the World Bank Group. </ul>
<p><p><br>
<center>
<a href="http://www.excite.com">
<img src="/Excite/pictures/AT-results_label.gif" BORDER=0
ALT="Results by Excite"></a></center>
</body></html>
