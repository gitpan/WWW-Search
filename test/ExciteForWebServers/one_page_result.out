http://www.worldbank.org/html/edi/sourcebook/sbxb13.htm
http://www.worldbank.org/html/edi/sourcebook/sbxa14.htm
http://www.worldbank.org/html/edi/sourcebook/sbxs11.htm
http://www.worldbank.org/html/edi/sourcebook/sbxa0104.htm
http://www.worldbank.org/html/prmge/tools/afrndx.htm
http://www.worldbank.org/html/prmge/afrndx.htm
http://www.worldbank.org/aftdr/gender/lstfrm.htm
http://www.worldbank.org/sp/sfunds/present/html/tsld0087.htm
http://www.worldbank.org/html/edi/iemt-pre/lopez.htm
http://www.worldbank.org/html/edi/iemt/lopez.htm
http://www.worldbank.org/html/fpd/technet/mdf/edi-trng/lopez.htm
http://www.worldbank.org/html/extdr/eligible.htm
http://www.worldbank.org/html/extpb/abshtml/13807.htm
http://www.worldbank.org/aftdr/afcbf/staff.htm
http://www.worldbank.org/html/extdr/medguide/madavo.htm
http://www.worldbank.org/aftdr/afcbf/proj.htm
http://www.worldbank.org/html/opr/ois/casp.html
http://www.worldbank.org/aftdr/particip/teams.htm
http://www.worldbank.org/html/extpb/abshtml/12994.htm
http://www.worldbank.org/html/extdr/offrep/afr/afrid97t.htm
